dotCMS/src/integration-test/java/com/dotcms/vanityurl/business/VanityUrlAPITest.java

```diff
@@ -93,35 +93,34 @@ public void getWorkingVanityUrl() {
             int action = 200;
             int order = 1;
 
-            contentlet1 = filtersUtil.createVanityUrl(title, site, uri,
-                    forwardTo, action, order, defaultLanguageId);
+            contentlet1 = new VanityUrlDataGen()
+                .title(title)
+                .site(site)
+                .uri(uri)
+                .forwardTo(forwardTo)
+                .action(action)
+                .order(order)
+                .language(defaultLanguage.getId())
+                .nextPersisted();
 
-            VanityUrl vanity1 = vanityUrlAPI.getVanityUrlFromContentlet(contentletAPI.find(contentlet1.getInode(), user, false));
+            VanityUrl vanity1 = vanityUrlAPI.fromContentlet(contentletAPI.find(contentlet1.getInode(), user, false));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'vanity1' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5434315)